docusaurus/docs/how-to-guides/panel-plugins/error-handling-for-panel-plugins.md

```diff
@@ -54,24 +54,6 @@ try {
 
 :::note
 
-Grafana displays the exception message in the UI as written, so use grammatically correct sentences. For more information, refer to the [Documentation style guide](https://grafana.com/docs/writers-toolkit/).
+Grafana displays the exception message in the UI as written, so use grammatically correct sentences. For more information, refer to the [documentation style guide](https://grafana.com/docs/writers-toolkit/).
 
 :::
-
```

Comment:
this information looks valuable. why are we removing it?

Reply:
i believe the content is related to data source plugins, so it is included in a separate file in the data source reorg PR. i agree that by splitting the error handling topics into 2 separate entities, we are creating content that could probably benefit from improvement. let's do a separate PR later to improve these, and perhaps add some info back that's specifially relevvant to panel plugins